.cirrus.yml

```diff
@@ -0,0 +1,251 @@
+env:
+  LANG: C
+  DEFAULT_CONFIG_OPTS: --enable-debug --without-libstatgrab
```

Comment:
Could you add --disable-dependency-tracking while you're at it? That shave about 30 seconds of a build on my machine.